### ISSUE TITLE
BatchNorm checkpoints are broken with the latest master

### ISSUE THREAD

**Issue by soumith** [pytorch/pytorch#5881]
https://github.com/pytorch/pytorch/pull/5766 broke loading of old checkpoints.
Fix incoming

**Comment by SsnL** [comment]
oh... is it what broke onnx test?

**Comment by soumith** [comment]
closed via https://github.com/pytorch/pytorch/pull/5892
